test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2417,6 +2417,7 @@ TEST_P(LocalityMapTest, UpdateMap) {
   }
   const double kErrorTolerance = 0.2;
   for (size_t i = 0; i < 3; ++i) {
+    gpr_log(GPR_INFO, "Locality %zu rete %f", i, locality_picked_rates[i]);
```

Comment:
I don't think the "%z" is portable.  You should instead use `PRIuPTR`, as is done elsewhere in our code.